Packs/CommonPlaybooks/.pack-ignore

```diff
@@ -115,3 +115,6 @@ ignore=RM108
 
 [file:playbook-Get_File_Sample_By_Hash_-_Generic_v2_README.md]
 ignore=RM108
+
+[file:playbook-Containment_Plan_-_Block_Indicators.yml]
+ignore=PB118
```

Comment:
explain about the ignore

Reply:
[PB118] - Playbook Containment Plan - Block Indicators contains inputs that are not used in any of its tasks: FilePath
If I remove this input from the playbook, it will break the containment plan playbook.